app/controllers/concerns/foreman_openscap/hosts_controller_extensions.rb

```diff
@@ -1,5 +1,11 @@
 module ForemanOpenscap
   module HostsControllerExtensions
+    extend ActiveSupport::Concern
+
+    included do
```

Comment:
The whole module is [prepended, not included](https://github.com/theforeman/foreman_openscap/blob/master/lib/foreman_openscap/engine.rb#L208), so I do not think this will work.

Reply:
@xprazak2 Any suggestions on how to use the before_action in a module that is prepended and not included?  I think I also have to add `define_action_permission` because if I put this in `select_multiple_openscap_proxy`:

```
    def select_multiple_openscap_proxy
      @hosts = find_multiple
      @hosts
    end
```

I end up with this error:

```
2018-05-15 12:26:22 79b673c6 [app] [W] Something went wrong while selecting hosts - ERF42-5531 [Foreman::Exception]: unknown permission for hosts#select_multiple_openscap_proxy
 | Foreman::Exception: ERF42-5531 [Foreman::Exception]: unknown permission for hosts#select_multiple_openscap_proxy
 | /usr/share/foreman/app/controllers/application_controller.rb:151:in `action_permission'
 | /usr/share/foreman/app/controllers/concerns/foreman/controller/action_permission_dsl.rb:19:in `action_permission'
 | /usr/share/foreman/app/controllers/concerns/application_shared.rb:22:in `current_permission'
 | /usr/share/foreman/app/controllers/hosts_controller.rb:660:in `resource_base'
 | /usr/share/foreman/app/controllers/hosts_controller.rb:742:in `find_multiple'
```

And the modal form ends up with the contents of hosts index view.